sklearn/preprocessing/_function_transformer.py

```diff
@@ -23,6 +23,8 @@ class FunctionTransformer(BaseEstimator, TransformerMixin):
 
     .. versionadded:: 0.17
 
+    Read more in the :ref:`User Guide <preprocessing>`.
```

Comment:
I would be great to link to the section that actually describes the function transformer, not just the preprocessing. you can just add a new tag to the preprocessing user guide.